src/pages/payment-methods/index.tsx

```diff
@@ -171,6 +173,36 @@ const DisplayAccordion = ({ locale }: PaymentMethodsProps) => {
     const { is_p2p_allowed_country } = React.useContext(DerivStore)
     const { is_eu } = useCountryRule()
     const [is_mobile] = useBrowserResize(992)
+    const { data, send } = useWS('crypto_config')
+    const [payment_method_data, setPaymentMethodData] = useState(payment_data)
+
+    // Here we send the request to the server on the first render of the page to get the data.
+    useEffect(() => {
+        send({})
+    }, [send])
+
+    useEffect(() => {
+        // First we check if the `data` exists or not, Then we manipulate the local data with the response from the server.
+        if (data) {
+            // Here we map over the local `payment_data` variable to manipulate it with the server response.
+            const updated_payment_data = payment_data.map((pdata) => {
```

Comment:
```suggestion
            const updated_payment_data = payment_data.map((paymentData) => {
```
@fasihali-deriv Can we have a better naming instead of `pdata` here? Like one I suggest or anything you think can make it more clear thanks 🙏